src/server/on-off-farm/controller.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * The first question to start the journey.
+ * @satisfies {Partial<ServerRoute>}
+ */
+export const onOffFarmGetController = {
+  handler(req, h) {
+    req.yar.set('animal', 'cow')
+    const onOffFarm = req.yar.get('onOffFarm')
+
+    return h.view('on-off-farm/index', {
+      pageTitle: 'Are you moving the cattle on or off your farm?',
+      heading: 'Are you moving the cattle on or off your farm?',
+      onOffFarm: {
+        value: onOffFarm
+      }
+    })
+  }
+}
+
+export const onOffFarmPostController = {
+  handler(req, res) {
+    const { onOffFarm } = req.payload
+
+    if (!onOffFarm) {
+      return res.view('on-off-farm/index', {
+        pageTitle: 'Are you moving the cattle on or off your farm?',
+        heading: 'Are you moving the cattle on or off your farm?',
+        errorMessage: {
+          text: 'Select if you are moving cattle on or off your farm?'
+        }
+      })
+    }
+
+    req.yar.set('onOffFarm', onOffFarm)
+
+    return res.view('on-off-farm/index', {
+      pageTitle: 'Are you moving the cattle on or off your farm?',
+      heading: 'Are you moving the cattle on or off your farm?',
+      onOffFarm: {
+        value: onOffFarm
+      }
+    })
+  }
+}
```

Comment:
Do we have a generic fallback on error that makes the journey recoverable for users?

Reply:
there is a generic 500 page yes